integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDBOperator.java

```diff
@@ -0,0 +1,740 @@
+        "ClusterFileStore@managed-server2 is NOT migrated to managed-server1");
+    logger.info("ClusterFileStore@managed-server2 is migrated to managed-server1");
+
+    assertTrue(checkStoreRuntime("ClusterJdbcStore@managed-server2",
+        "managed-server1"),
+        "JdbcStore@managed-server2 is NOT migrated to managed-server1");
+    logger.info("JdbcStore@managed-server2 is migrated to managed-server1");
+
+    assertTrue(checkJtaRecoveryServiceRuntime("managed-server1",
+        "managed-server2", "true"), "JTA RecoveryService@managed-server2 is not migrated to managed-server1");
+    logger.info("JTA RecoveryService@managed-server2 is migrated to managed-server1");
+
+    runJmsClientOnAdminPod("receive",
+        "ClusterJmsServer@managed-server2@jms.testUniformQueue");
+    runJmsClientOnAdminPod("receive",
+        "JdbcJmsServer@managed-server2@jms.jdbcUniformQueue");
+
+    // Restart the managed server(2) to make sure the JTA Recovery Service is
+    // migrated back to original hosting server
+    restartManagedServer("managed-server2");
+    assertTrue(checkJtaRecoveryServiceRuntime("managed-server2",
+        "managed-server2", "true"),
+        "JTARecoveryService@managed-server2 is not on managed-server2 after restart");
+    logger.info("JTA RecoveryService@managed-server2 is migrated back to managed-server1");
+    assertTrue(checkJtaRecoveryServiceRuntime("managed-server1",
+        "managed-server2", "false"),
+        "JTARecoveryService@managed-server2 is not deactivated on managed-server1 after restart");
+    logger.info("JTA RecoveryService@managed-server2 is deactivated on managed-server1 after restart");
+
+    assertTrue(checkStoreRuntime("ClusterFileStore@managed-server2",
+        "managed-server2"),
+        "FileStore@managed-server2 is NOT migrated back to managed-server2");
+    logger.info("FileStore@managed-server2 is migrated back to managed-server2");
+    assertTrue(checkStoreRuntime("ClusterJdbcStore@managed-server2",
+        "managed-server2"),
+        "JdbcStore@managed-server2 is NOT migrated back to managed-server2");
+    logger.info("JdbcStore@managed-server2 is migrated back to managed-server2");
+
+    assertTrue(checkJmsServerRuntime("ClusterJmsServer@managed-server2",
+        "managed-server2"),
+        "ClusterJmsServer@managed-server2 is NOT migrated back to to managed-server2");
+    logger.info("ClusterJmsServer@managed-server2 is migrated back to managed-server2");
+    assertTrue(checkJmsServerRuntime("JdbcJmsServer@managed-server2",
+        "managed-server2"),
+        "JdbcJmsServer@managed-server2 is NOT migrated back to managed-server2");
+    logger.info("JdbcJmsServer@managed-server2 is migrated back to managed-server2");
+  }
+
+  /**
+   * Uninstall DB operator.
+   * The cleanup framework does not uninstall storageclass.
+   * Do it here for now.
+   */
+  @AfterAll
+  public void tearDownAll() throws ApiException {
+    DbUtils.deleteStorageclass();
```

Comment:
use static import for deleteStorageclass.
Also you may want to add the check for "SKIP_CLEANUP". If the SKIP_CLEANUP is set to true, you may not want to delete it.

Reply:
Done.